JWTDecodeTests/JWTDecodeSpec.swift

```diff
@@ -410,11 +410,7 @@ class JWTDecodeSpec: XCTestCase {
     }
 }
 
-#if compiler(>=6.0)
-extension JWTDecodeError: @retroactive Equatable {}
-#else
 extension JWTDecodeError: Equatable {}
```

Comment:
In this case, the `@retroactive` declaration should be left in place, because it will always be built with the Swift 6.0 compiler (or later).

Reply:
See the warnings below:

<img width="1443" alt="Screenshot 2025-04-23 at 15 01 16" src="https://github.com/user-attachments/assets/f9bd7bd1-d708-426c-ae18-768eb04e4734" />